metadata-ingestion/src/datahub/ingestion/source/identity/okta.py

```diff
@@ -364,18 +357,11 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
             for user_count, datahub_corp_user_snapshot in enumerate(
                 datahub_corp_user_snapshots
             ):
-                # Add GroupMembership aspect populated in Step 2 if applicable.
-                if (
-                    datahub_corp_user_snapshot.urn
-                    in datahub_corp_user_urn_to_group_membership
-                ):
-                    datahub_group_membership = (
-                        datahub_corp_user_urn_to_group_membership.get(
-                            datahub_corp_user_snapshot.urn
-                        )
-                    )
-                    assert datahub_group_membership is not None
-                    datahub_corp_user_snapshot.aspects.append(datahub_group_membership)
+                # Add GroupMembership aspect populated in Step 2.
+                datahub_corp_user_urn: str = datahub_corp_user_snapshot.urn
+                datahub_group_membership: GroupMembershipClass = datahub_corp_user_urn_to_group_membership[datahub_corp_user_urn]
```

Comment:
this code looks eerily similar to the azure code above. we probably want to refactor it in the future

Reply:
Yeah I know...I'll write a TODO here